test-it/ViaTests/UnflattenJSONWithNullValues.via

```diff
@@ -0,0 +1,193 @@
+define (MyVI dv(.VirtualInstrument (
```

Comment:
Extensive testing looks good. I would also add a few tests where "path" is also passed. If added in the VI test suite, that is good enough.

Reply:
I'll add it to the VI test suite